packages/www/docs/guides/upload-a-video/index.mdx

```diff
@@ -294,56 +296,13 @@ const response = await axios
 
 Response
 
-- When the video is uploaded, the result should return a 200 response
+When the video is uploaded, the result should return a `200 OK` response with an
+empty body.
 
-```json
-{
-  "id": "4c21895e-2488-4c0a-9377-3369ff586ddd",
-  "hash": [
-    {
-      "hash": "6dad0a8e7a92d537c6305fe6ae1e1881",
-      "algorithm": "md5"
-    },
-    {
-      "hash": "6132f4e43472b56b4047b7ad018147056c0c2dba9306cd04504e9c8208e72257",
-      "algorithm": "sha256"
-    }
-  ],
-  "name": "Waterfall",
-  "size": 32407849,
-  "status": "ready",
-  "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
-  "createdAt": 1652901725324,
-  "updatedAt": 1652901778931,
-  "videoSpec": {
-    "format": "mp4",
-    "tracks": [
-      {
-        "fps": 23.976023976023978,
-        "type": "video",
-        "codec": "h264",
-        "width": 4096,
-        "height": 2048,
-        "bitrate": 21324821,
-        "duration": 12.011667,
-        "pixelFormat": "yuv420p"
-      },
-      {
-        "type": "audio",
-        "codec": "aac",
-        "bitrate": 253375,
-        "channels": 2,
-        "duration": 12.031667,
-        "sampleRate": 48000
-      }
-    ],
-    "duration": 12.031667
-  },
-  "playbackId": "4c21k7z2tjcdg8wt",
-  "playbackUrl": "https://livepeercdn.com/recordings/54aeff83-404c-490a-9afa-99296a932401/index.m3u8",
-  "downloadUrl": "https://livepeercdn.com/asset/4c21k7z2tjcdg8wt/video"
-}
-```
+- If you want to track the status of your upload, you can use the `task`
+  endpoint by passing the `task.id` you received during the previous request. To
+  retrieve the data of your uploaded asset, you can use the `asset` endpoint by
+  passing the `asset.id` as a parameter.
```

Comment:
Would explain first how to track it without dealing with tasks, using only the `asset.id`, and then in another point say "you can also track partial progress by using the task API etc..."

Also, we should link to the corresponding API references pages for the APIs mentioned here (GET asset and task).

Reply:
Good point, added this to the new pr #1154